psi4/driver/p4util/util.py

```diff
@@ -138,13 +138,12 @@ def set_memory(inputval):
         units = ''
     # Handle memory given as a string
     if isinstance(inputval, basestring):
```

Comment:
The builtin `basestring` has been removed in python 3. This will throw:

```python
NameError  File "/Users/admin/Workspace/Git/psi4/objdir/stage/Users/admin/psi4-dev-installs/test/lib/psi4/driver/p4util/util.py", line 140, in set_memory
    if isinstance(inputval, basestring):
NameError: name 'basestring' is not defined
```

Which points out another problem; `psi4.set_memory()` Should be checked somewhere in quick-tests, but we have python3 builds on travis and this PR passed.

Also, should probably add a `psi4.set_memory()` call somewhere in quick-tests that uses a string as input.

Possible Solution (I haven't tested this)
- Just check if input is `float` or `int` if not str is assumed and any non-`str` inputs will lead to exceptions raising in `re.search`